{{ cookiecutter.repo_name }}/Dockerfile

```diff
@@ -13,4 +14,9 @@ RUN pip install -r /home/drtools/requirements.txt && \
 COPY . /home/drtools/{{ cookiecutter.repo_name }}
 RUN pip install -e /home/drtools/{{ cookiecutter.repo_name }}
 
+# Copy configurations into place if needed
+COPY conf/dask-config.yml /home/drtools/.dask/config.yml
+COPY conf/logging.cfg /etc/luigi/luigi-logging.cfg
+COPY conf/luigi.cfg /etc/luigi/client.cfg
```

Comment:
I am always confused about the name and correct location of this file. But this seems to be a good solution: name it `luigi.cfg` in the template, but `client.cfg` in the image. Well, it's also somehow confusing, but I can't see a better option. :+1: